.github/workflows/daily-update.yml

```diff
@@ -42,9 +42,9 @@ jobs:
           git submodule init
           git submodule update --recursive
           cd Wippersnapper_Boards
-          git reset --hard origin/rp2040_datalogger_feather
+          git reset --hard origin/offline-mode
           cd ../Wippersnapper_Components
-          git reset --hard origin/main
```

Comment:
Double check old branches versus new ones, for any missing boards / settings / components